lib/constants.ex

```diff
@@ -18,6 +24,9 @@ defmodule Constants do
   @spec domain_voluntary_exit() :: <<_::32>>
   def domain_voluntary_exit, do: <<4, 0, 0, 0>>
 
+  @spec domain_bls_to_execution_change() :: <<_::32>>
```

Comment:
We should use `SszTypes.domain_type()` as the return type. Feel free to update the other `domain_*` functions